components/gitpod-db/src/typeorm/project-db-impl.ts

```diff
@@ -80,7 +80,15 @@ export class ProjectDBImpl extends TransactionalDBImpl<ProjectDB> implements Pro
 
         const queryBuilder = projectRepo
             .createQueryBuilder("project")
-            .where("project.cloneUrl LIKE :searchTerm", { searchTerm: `%${searchTerm}%` })
+            // TODO: should we make this conditional if searchTerm has a value to avoid the scan?
+            .where(
+                new Brackets((qb) => {
+                    qb.where("project.cloneUrl LIKE :searchTerm", { searchTerm: `%${searchTerm}%` }).orWhere(
+                        "project.name LIKE :searchTerm",
```

Comment:
It does not seem we have an index of project name but on clone URL? Do we need to add one? cc @geropl

Reply:
In this case I don't think an index would help since it has a leading wildcard in the search. It would be helpful for sorting by name though, which we would like to have on the new list UI.